sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -202,6 +237,12 @@ cdef class StateSampler(object):
     for state in self.scoped_states_by_name.values():
       state_msecs = int(1e-6 * state.nsecs)
       state.counter.update(state_msecs - state.counter.value())
+      if (<ScopedState>state).is_active:
+        state_msecs = int(1e-6 * state.active_nsecs)
```

Comment:
<!--new_thread; commit:c0086ba69497a5cbd3269f1d691831ca78563dcc; resolved:0-->
It seems error-prone to reuse the `state_msecs` variable here.  Can we just do something like

```python
active_state_msecs = 0
if is_active:
  active_state_msecs = int(1e-6 * state.active_nsecs)
[...]
```